History.md

```diff
@@ -2,6 +2,16 @@
 
 ## v1.7.1, TBD
 
+### Breaking changes
+N/A
+
+### Migration Steps
+* Update `@babel/runtime` npm package to version 7.0.0-beta.56 or later
+```sh
+    meteor npm install @babel/runtime@latest
```

Comment:
Let's indent the triple back-ticks by two spaces to line up with `Update`, and then de-indent the `meteor npm ...` command so that it starts on the same column (also two spaces from the margin).